libs/wingsdk/src/shared-aws/bucket.inflight.ts

```diff
@@ -252,7 +253,7 @@ export class BucketClient implements IBucketClient {
    * @param key The key to reach
    * @param duration Time until expires
    */
-  public async signed_url(key: string, duration?: Duration): Promise<string> {
+  public async signedUrl(key: string, duration?: Duration): Promise<string> {
```

Comment:
How come this didn't break any tests?

Reply:
I went to check, this method has not been implemented, it throws an exception.

I will fix this in the next few days